scripts/release.sh

```diff
@@ -0,0 +1,281 @@
+    cd ..
+fi
+# If, for soem reason, we are not now in the correct working directory, exit
+if [[ $PWD != *"sdl_ios" ]]; then
+    echo "Please run this from the sdl_ios project root or the sdl_ios/scripts directory"
+    exit 0
+fi
+
+
+# 1 bump version in projectFile
+echo
+echo "Step 1: Update version in Project File"
+
+# 1.1) get the current version and build from the podspec file
+project_file=./SmartDeviceLink-iOS.xcodeproj/project.pbxproj
+new_file=./SmartDeviceLink-iOS.xcodeproj/new.pbxproj
+current_version_number=$(sed -n '/MARKETING_VERSION/{s/MARKETING_VERSION = //;s/;//;s/^[[:space:]]*//;p;q;}' $project_file)
+current_build_number=$(sed -n '/CURRENT_PROJECT_VERSION/{s/CURRENT_PROJECT_VERSION = //;s/;//;s/^[[:space:]]*//;p;q;}' $project_file)
+echo "Current Version "$current_version_number
+echo "Current Build "$current_build_number
+
+# 1.2) prompt user for new version
+echo "Enter the new version number (semantic versioning x.x.x format) or blank to skip: "
+read new_version_number
+
+# 1.3) if not blank, or not the same
+if [ -z $new_version_number ]; then
+    echo "No version number entered. Skipping..."
+else
+    if [ $current_version_number != $new_version_number ]; then
+        echo "Changing Version Number in $project_file"
+        # 1.4) swap new version in to file
+        sed '/MARKETING_VERSION/{s/'$current_version_number'/'$new_version_number'/;}' $project_file > $new_file
+        mv -f $new_file $project_file
+    else
+        echo "No project file change needed for version"
+    fi
+fi
+
+# in the event that the build number somehow is not what it should be, we have code to fix it.
+#echo "New Build?"
+#read new_build_number
+# per review, the new build number will always be 1
+new_build_number=1
+if [ -z $new_build_number ]; then
+    echo "No project file change needed for build number"
+else
+    if [ $current_build_number != $new_build_number ]; then
+        echo make the change
+        # 1.4) swap new version in to file
+        sed '/CURRENT_PROJECT_VERSION/{s/'$current_build_number'/'$new_build_number'/;}' $project_file > $new_file
+        mv -f $new_file $project_file
+    else
+        echo "No project file change needed for build number"
+    fi
+fi
+
+
+# 2 update version in podspec
+# SmartDeviceLink-iOS.podspec
+echo
+echo "Step 2: Update version in podspec File"
+
+# 2.1) get the current version from the podspec file
+PodSpecFile=SmartDeviceLink-iOS.podspec
+PodSpecNewFile=NewFile.podspec
+CurrentVersion=$(sed -n '/s.version/{s/s.version//;s/=//;s/[\"]//g;s/^[[:space:]]*//g;p;q;}' $PodSpecFile)
+echo "Current Version="$CurrentVersion
+
+# 2.2) prompt user for new version
+#echo "New Version?"
+#read new_version
+# per review, the new version should be the same as above, but we should check with the user.
+echo "Enter the new version number for podfile (semantic versioning x.x.x format) or blank to set to match project file ("$new_version_number"): "
+read new_version
```

Comment:
Remove unused comments. As last review mentioned, this must be equal to the project version received above. Remove this request from the user.